classes/classes/Items/MutationsHelper.as

```diff
@@ -0,0 +1,48 @@
+package classes.Items 
+{
+	import classes.*;
+	
+	/**
+	 * Helper class to get rid of the copy&paste-mess from classes.Items.Mutations
+	 * @author Stadler76
+	 */
+	public class MutationsHelper extends BaseContent 
+	{
+		include "../../../includes/appearanceDefs.as";
+
+		public function MutationsHelper() 
+		{
+		}
+
+		public function restoreArms(changes:Number, changeLimit:Number, keepArms:Array = null):Number
```

Comment:
Like I said: restoreArms for starters and keepArms is for possible future use.
If this is accepted the next PR will likely be for remove feather hair.